zipline/sources/test_source.py

```diff
@@ -88,7 +87,7 @@ def advance_current(cur):
 
     # yield count trade events, all on trading days, and
     # during trading hours.
-    for i in range(count):
+    while cur < end:
```

Comment:
Making sure I'm reading this right. In the case of `start == end` like with the default parameters and no repeats, this will yield no data, correct?